atd-etl/app/process/socrata_queries.py

```diff
@@ -0,0 +1,86 @@
+"""
+Socrata Queries
+Author: Austin Transportation Department, Data and Technology Office
+
+Description: The purpose of this script is to define graphql query templates
+to run against the Hasura endpoint for Socrata upsertion.
+
+Important: These templates require importing the Template class from the string library.
+"""
+from string import Template
+
+# Queries Hasura to retrieve records for
+# https://data.austintexas.gov/d/y2wy-tgr5
+crashes_query_template = Template(
+    """
+    query getCrashesSocrata {
+        atd_txdot_crashes (limit: $limit, offset: $offset, order_by: {crash_id: asc}, where: {city_id: {_eq: 22}}) {
```

Comment:
just putting a bookmark here so you remember that if we change how we query by city id we'll need to modify this code.

Reply:
Awesome, I'll keep this in mind too if we decide to add more `city_id`s.